src/corefx/System.Globalization.Native/idna.cpp

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#include <stdint.h>
+#include <uchar.h>
+#include <unicode/uidna.h>
+
+const uint32_t AllowUnassigned = 0x1;
+const uint32_t UseStd3AsciiRules = 0x2;
```

Comment:
Maybe a question for @nguerrera, but what's our thinking on structuring these native libraries with regards to .h files?